lesson03/main.js

```diff
@@ -0,0 +1,20 @@
+const ul = document.getElementById("js-target");
+
+const li = [];
+const a = [];
+const img = [];
+
+for (let i=0; i < 2; i++){
```

Comment:
IMO
Numbers like 2, where you don't know what they represent, are called magic numbers and should be avoided. It is recommended to create an array first and specify it as follows.
```suggestion
for (let i=0; i < imgSrcs.length; i++){
```

Reply:
Thank you mai-san :)
I won't use magic numbers :)